docs/build-og-images.js

```diff
@@ -0,0 +1,116 @@
+const path = require('path');
+const fs = require('fs');
+const { createCanvas, loadImage, registerFont } = require('canvas');
+
+registerFont('static/font/Aeonik-Bold.otf', { family: 'Aeonik Bold' });
+
+const getMarkdownHeader = (path) => {
+  const content = fs.readFileSync(path, 'utf-8');
+  const headers = content
+    .split('\n')
+    .map((line) => line.trim())
+    .filter((line) => line.startsWith('#'))
+    .map((line, index) => ({
+      level: line.match(/#/g).length,
+      title: line.replace(/#/g, '').trim(),
+      index,
+    }))
+    .sort((a, b) => a.level - b.level || a.index - b.index);
+
+  return headers[0]?.title || '';
+};
```

Comment:
it might be simpler to just find a line that starts with a single #

Reply:
yes, but it take into account situations in which a single # will not be present